builder/hyperv/common/step_polling_installation.go

```diff
@@ -15,10 +15,10 @@ import (
 
 const port string = "13000"
 
-type StepPollingInstalation struct {
+type StepPollingInstallation struct {
```

Comment:
Come to think of it, it looks like this file isn't even in use in Packer anymore.  @taliesins should I just delete the file? StepPollingInstalation isn't used.